functions/scheduled.js

```diff
@@ -3,10 +3,74 @@ const admin = require("firebase-admin");
 const functions = require("firebase-functions");
 const { HttpsError } = require("firebase-functions/lib/providers/https");
 
-const { chunk, sampleSize } = require("lodash");
+const { chunk, sampleSize, flatMap } = require("lodash");
 const hoursToMilliseconds = require("date-fns/hoursToMilliseconds");
 
 const DEFAULT_RECENT_USERS_IN_VENUE_CHUNK_SIZE = 6;
+const SECTION_PREVIEW_USER_DISPLAY_COUNT = 14;
+
+exports.updateSeatedUsersCountInAuditorium = functions.pubsub
+  .schedule("every 5 seconds")
```

Comment:
Reduce it to 5 minutes to reduce the costs

Reply:
Btw, in this exact scheduled function when you fetch seated users you can delete the old ones

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2318/commits/31f0c55b5db370cbcbe5998ea7ab45d54b0916fb

Reply:
we will cope with "dangling" users in a separate PR